Gemfile

```diff
@@ -2,7 +2,8 @@ source 'https://rubygems.org'
 
 gem 'rspec'
 gem "rack"
-gem "sinatra"
+gem "sinatra", "~> 2.0.0.rc2"
```

Comment:
@perlun sure, I will agree. thanks for the mentioning. I think better solution is to test both Rack 1 and Rack 2 on Travis. I will change files for that. Just moment. 

@dentarg maybe I will prepare another gemfile such as `gemfiles/Gemfile.rack1`, or write `if condition` to `Gemfile`.